website/content/specification/computer-example.xml

```diff
@@ -164,8 +206,59 @@
+        </define-flag>
+        <define-flag name="uuid" as-type="uuid">
+            <formal-name>Property Universally Unique Identifier</formal-name>
+            <!-- identifier declaration -->
+            <description>A unique identifier for a property.</description>
+        </define-flag>
+        <define-flag name="ns" as-type="uri" default="http://csrc.nist.gov/ns/metaschema/computer/example">
+            <formal-name>Property Namespace</formal-name>
+            <description>A namespace qualifying the property's name. This allows different organizations to associate distinct semantics with the same name.</description>
+            <remarks>
+                <p>This value must be an <a href="https://datatracker.ietf.org/doc/html/rfc3986#section-4.3">absolute URI</a> that serves as a naming system identifier.</p>
+                <p>When a <code>ns</code> is not provided, its value should be assumed to be <code>http://csrc.nist.gov/ns/oscal</code> and the name should be a name defined by the associated OSCAL model.</p>
+            </remarks>
+        </define-flag>
+        <define-flag name="value" as-type="string" required="yes">
+            <formal-name>Property Value</formal-name>
+            <description>Indicates the value of the attribute, characteristic, or quality.</description>
+        </define-flag>
+        <define-flag name="class" as-type="token">
+            <formal-name>Property Class</formal-name>
+            <description>A textual label that provides a sub-type or characterization of the
+                property's <code>name</code>.</description>
+            <remarks>
+                <p>This can be used to further distinguish or discriminate between the semantics of multiple properties of the same object with the same <code>name</code> and <code>ns</code>, or to group properties into categories.</p>
+                <p>A <code>class</code> can be used in validation rules to express extra constraints over named items of a specific <code>class</code> value. It is available for grouping, but unlike <code>group</code> is not expected specifically to designate any group membership as such.</p>
+            </remarks>
+        </define-flag>
+        <define-flag name="group" as-type="token">
+            <formal-name>Property Group</formal-name>
+            <description>An identifier for relating distinct sets of properties.</description>
+            <remarks>
+                <p>Different sets of properties may relate to separate contexts. Declare a group on a property to associate it with one or more other properties in a given context.</p>
+            </remarks>
+        </define-flag>
+        <model>
+            <field ref="remarks"/>
+        </model>
+        <constraint>
+            <allowed-values target=".[not(@ns) or @ns=('http://csrc.nist.gov/ns/metaschema/computer/example')]/@name">
```

Comment:
Blocking: should we make this align with the schema URI (or change `namespace` but I can't suggest that with the GitHub PR suggestion interface).

```suggestion
            <allowed-values target=".[not(@ns) or @ns=('http://example.com/ns/computer')]/@name">
```